Public/Invoke-DockerPull.ps1

```diff
@@ -0,0 +1,32 @@
+function Invoke-DockerPull {
+    [CmdletBinding()]
+    param (
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Registry,
+        [Parameter(mandatory=$true)]
+        [String]
+        $Image,
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Tag = 'latest',
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Digest = ''
+    )
+
+    $RegistryPostfixed = Add-RegistryPostfix -Registry $Registry
+
+    # Pulls by tag by default
+    $ImageToPull = "${RegistryPostfixed}${Image}:${Tag}"
+
+    # Digest overrides tag however
+    if (-Not [String]::IsNullOrEmpty($Digest)) {
```

Comment:
Believe you can "just" do:

```suggestion
    if (!$Digest) {
```

Reply:
Thanks. Good to know that `-Not` became `!`.  Used to be not the case.

Reply:
The change won't work correctly. Reverting to old method.